libiocage/lib/Jail.py

```diff
@@ -143,7 +152,15 @@ def start(self):
 
         release = self.release
 
-        backend = None
+        backend: Union[
+            libiocage.lib.StandaloneJailStorage.StandaloneJailStorage,
+            libiocage.lib.NullFSBasejailStorage.NullFSBasejailStorage,
+            libiocage.lib.ZFSBasejailStorage.ZFSBasejailStorage,
+            None,
+        ] = None
+
+        if self.config["basejail_type"] == "standalone":
+            backend = libiocage.lib.StandaloneJailStorage.StandaloneJailStorage
```

Comment:
```
libiocage/lib/Jail.py:163: error: Incompatible types in assignment (expression has type Type[StandaloneJailStorage], variable has type "Union[StandaloneJailStorage, NullFSBasejailStorage, ZFSBasejailStorage, None]")
```

Reply:
Why is it `type Type[StandaloneJailStorage]`, but not `type "Type[Union[StandaloneJailStorage, ...."`